`performance_counters.hpp` gives warning C5054: operator '-': deprecated between enumerations of different types

Compile RC_2_0 Branch with MSVC 19.42.34436.0 & C++20

`D:\a\qBittorrent\libtorrent\include\libtorrent/performance_counters.hpp(463): warning C5054: operator '-': deprecated between enumerations of different types`

https://github.com/arvidn/libtorrent/blob/c31d90b59ffb3910b6d36ba7ad914e0e489d2b6c/include/libtorrent/performance_counters.hpp#L463
____

Compile RC_1_2 Branch with MSVC 19.42.34436.0 & C++20

`D:\a\qBittorrent\libtorrent\include\libtorrent/performance_counters.hpp(473): warning C5054: operator '-': deprecated between enumerations of different types`

https://github.com/arvidn/libtorrent/blob/2e537ee7afcfa92d02862d81c53db2a363bfdd1e/include/libtorrent/performance_counters.hpp#L473

--- a/include/libtorrent/performance_counters.hpp
+++ b/include/libtorrent/performance_counters.hpp
@@ -460,7 +460,7 @@ namespace libtorrent {
 			num_queued_tracker_announces,
 
 			num_counters,
-			num_gauges_counters = num_counters - num_stats_counters
+			num_gauges_counters = num_counters - static_cast<int>(num_stats_counters)
 		};
 #ifdef ATOMIC_LLONG_LOCK_FREE
 #define TORRENT_COUNTER_NOEXCEPT noexcept

